MAUI/Cartesian-Charts/Axis/Axislabels.md

```diff
@@ -166,4 +166,79 @@ chart.XAxes.Add(primaryAxis);
 
 {% endhighlight %}
 
+{% endtabs %}
+
+## Smart Axis Labels
+
+Axis labels may overlap with each other based on chart dimensions and label size. The [LabelsIntersectAction]() property of axis is used to avoid overlapping of axis labels. The default value of the LabelsIntersectAction is `None`; other available values are `MultipleRows`, `Hide`, and `Wrap`.
```

Comment:
@Sowndharya1911 update enum values with hyperlink. Will include API links once it is published